Source/Magick.NET.CrossPlatform/ubuntu.16.04/authorized_keys

```diff
@@ -0,0 +1 @@
+ssh-rsa <your public key here>
```

Comment:
Can we remove this file from the repo and add it to the ignore list so I don't check it in by accident?